### ISSUE TITLE
Registry error: OCI API error: manifest unknown

### ISSUE THREAD

**Issue by kravciak** [kubewarden/allow-privilege-escalation-psp-policy#48]
### Current Behavior
version: psp-allow-privilege-escalation:v0.2.4

```
# Policy server log
ERROR policy_server: error while downloading policy clusterwide-test-allow-privilege-escalation-psp from ghcr.io/kubewarden/policies/psp-allow-privilege-escalation:v0.2.4: 
the policy registry://ghcr.io/kubewarden/policies/psp-allow-privilege-escalation:v0.2.4 could not be downloaded due to error: 
Registry error: url https://ghcr.io/v2/kubewarden/policies/psp-allow-privilege-escalation/manifests/v0.2.4, envelope: OCI API errors: [OCI API error: manifest unknown]

# Trying manually
~ kwctl pull ghcr.io/kubewarden/policies/psp-allow-privilege-escalation:v0.2.4
Error: the policy registry://ghcr.io/kubewarden/policies/psp-allow-privilege-escalation:v0.2.4 could not be downloaded due to error: 
Registry error: url https://ghcr.io/v2/kubewarden/policies/psp-allow-privilege-escalation/manifests/v0.2.4, envelope: OCI API errors: [OCI API error: manifest unknown]
```

**Comment by flavio** [comment]
@kravciak , @jordojordo I've pushed the fix and tagged a new release. The 0.2.5 is going to address this issue

**Comment by kravciak** [comment]
Fixed in `v0.2.5`, thanks
